unity-renderer/Assets/UIComponents/Scripts/Components/ColorPicker/ColorSelector.cs

```diff
@@ -30,7 +30,9 @@ public void Populate(IReadOnlyList<Color> colors)
     public void Cleanup()
     {
         foreach (var colorToggle in colorToggles)
-            DestroyImmediate(colorToggle);
+            Destroy(colorToggle.gameObject);
```

Comment:
Test thoroughly that this is not being called in a test. otherwise we will need check `#if UNITY_EDITOR` and if we are in play mode to use DestroyImmediate instead.

Reply:
You're completely right... I didn't take into account the use of this logic in editor mode where we want to make use of the [REFRESH] button in the UI Component inspector. So I have added the `#if UNITY_EDITOR` directive to take into account that scenario 👍 Done here: https://github.com/decentraland/unity-renderer/pull/5136/commits/a3e9d0f723a77703b57e1d0c9361210981cfd13c